schedule/store.go

```diff
@@ -115,6 +129,63 @@ func (store *Store) FindMany(ctx context.Context, ids []string) ([]Schedule, err
 
 	return result, nil
 }
+
+func (store *Store) FindMany(ctx context.Context, ids []string) ([]Schedule, error) {
+	return store.FindManyTx(ctx, nil, ids)
+}
+
+func (store *Store) FindManyByAssignments(ctx context.Context, tx *sql.Tx, userID string, rotationIDs []string) ([]Schedule, error) {
+	err := permission.LimitCheckAny(ctx, permission.All)
+	if err != nil {
+		return nil, err
+	}
+
+	err = validate.UUID("UserID", userID)
+	if err != nil {
+		return nil, err
+	}
+	uid, err := uuid.Parse(userID)
+	if err != nil {
+		return nil, err
+	}
+
+	var rids = []uuid.UUID{}
+	for _, r := range rotationIDs {
+		rid, err := uuid.Parse(r)
+		if err != nil {
+			return nil, err
+		}
+		rids = append(rids, rid)
+	}
+
+	rows, err := gadb.New(store.db).FindManyByAssignments(ctx, gadb.FindManyByAssignmentsParams{
```

Comment:
Use/require the DB to be passed into this method -- as it is it accepts a transaction but uses the connection pool directly (could trigger a deadlock scenario if the pool is exhausted)